blog/2024/release-updates.adoc

```diff
@@ -0,0 +1,26 @@
+:title: Backwards compatibility in Keycloak releases
+:date: 2024-08-19
+:publish: false
+:author: Stian Thorgersen
+
+There will not be a Keycloak 27.0 release, at least for some time. We are certainly not going to stop bringing new
+features and enhancements to Keycloak, but we are committed to doing so in a backwards compatible way. Hence, the
+release following Keycloak 26.0 will be 26.1 instead.
```

Comment:
I'd prefer the blog post starting with the why we're going for minor releases. While this this sentence grabs all of the attention and is quite a cliffhanger, I'd prefer the why first..